.gitattributes

```diff
@@ -1,2 +1,7 @@
 # Normalize EOL for all files that Git considers text files.
 * text=auto eol=lf
+
+# Only include the addons folder when downloading from the Asset Library.
+/**                   export-ignore
+/addons/block_code    !export-ignore
+/addons/block_code/** !export-ignore
```

Comment:
This actually causes no files to be included. You can test it with:

```
git archive --worktree-attributes --format tar HEAD | tar -t
```

`--worktree-attributes` means that uncommitted changes to `.gitattributes` will be used; it's just for easier development.


I'll push an alternative commit.